x-pack/plugins/cases/public/components/connectors/jira/case_fields.test.tsx

```diff
@@ -122,15 +122,12 @@ describe('Jira Fields', () => {
         connector={connector}
       />
     );
-    expect(wrapper.find('[data-test-subj="card-list-item"]').at(0).text()).toEqual(
-      'Issue type: Task'
-    );
-    expect(wrapper.find('[data-test-subj="card-list-item"]').at(1).text()).toEqual(
-      'Parent issue: Parent Task'
-    );
-    expect(wrapper.find('[data-test-subj="card-list-item"]').at(2).text()).toEqual(
-      'Priority: High'
-    );
+
+    const nodes = wrapper.find('[data-test-subj="card-list-item"]').hostNodes();
```

Comment:
Enzyme returns both React components and DOM nodes. This mess up with the order of the nodes. `hostNodes` returns only the DOM nodes.